profiles/Ashish-Khanagwal.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/75534912/189076823-8aeefe43-1138-4837-8766-077da3817974.jpg",
+  "issueId": 1368,
+  "name": "Ashish Khanagwal",
+  "username": "Ashish-Khanagwal"
+}
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

Reply:
Okay, I'll take care of it

Reply:
missing endline added, you can review now. If need further more changes then ping me.
Thank you